### ISSUE TITLE
Install failure with GKE Autopilot

### ISSUE THREAD

**Issue by crcsmnky** [signalfx/splunk-otel-collector-chart#555]
Using the following values with `helm`:
```
clusterName: autpilot-1
cloudProvider: "gcp"
distribution: "gke/autopilot"
splunkObservability:
  realm: us1
  accessToken: *********
```

And installed via:
`> helm install splunk-otel --values autopilot-values.yaml splunk-otel-collector-chart/splunk-otel-collector`

This was the output (formatting mine):
```
Error: INSTALLATION FAILED: admission webhook "gkepolicy.common-webhooks.networking.gke.io" denied the request: GKE Policy Controller rejected the request because it violates one or more policies: 

{
  "[denied by autogke-disallow-hostnamespaces]": [
    "enabling hostNetwork is not allowed in Autopilot. Requested by user: '**********', groups: 'system:authenticated'."
  ],
  "[denied by autogke-no-host-port]": [
    "container otel-collector specifies a host port; disallowed in Autopilot. Requested by user: '**********', groups: 'system:authenticated'."
  ],
  "[denied by autogke-no-write-mode-hostpath]": [
    "hostPath volume host-dev used in container otel-collector uses path /dev which is not allowed in Autopilot. Allowed path prefixes for hostPath volumes are: [/var/log/]. Requested by user: '**********', groups: 'system:authenticated'.",
    "hostPath volume host-etc used in container otel-collector uses path /etc which is not allowed in Autopilot. Allowed path prefixes for hostPath volumes are: [/var/log/]. Requested by user: '**********', groups: 'system:authenticated'.",
    "hostPath volume host-proc used in container otel-collector uses path /proc which is not allowed in Autopilot. Allowed path prefixes for hostPath volumes are: [/var/log/]. Requested by user: '**********', groups: 'system:authenticated'.",
    "hostPath volume host-run-udev-data used in container otel-collector uses path /run/udev/data which is not allowed in Autopilot. Allowed path prefixes for hostPath volumes are: [/var/log/]. Requested by user: '**********', groups: 'system:authenticated'.",
    "hostPath volume host-sys used in container otel-collector uses path /sys which is not allowed in Autopilot. Allowed path prefixes for hostPath volumes are: [/var/log/]. Requested by user: '**********', groups: 'system:authenticated'.",
    "hostPath volume host-var-run-utmp used in container otel-collector uses path /var/run/utmp which is not allowed in Autopilot. Allowed path prefixes for hostPath volumes are: [/var/log/]. Requested by user: '**********', groups: 'system:authenticated'."
  ]
}
```

Chart used:
```
> helm list
NAME       	NAMESPACE	REVISION	UPDATED                            	STATUS	CHART                       	APP VERSION
splunk-otel	default  	1       	2022-10-06 16:42:21.58626 -0700 PDT	failed	splunk-otel-collector-0.59.0	0.59.1
```

**Comment by atoulme** [comment]
Sorry to hear you ran into trouble. We have some additional steps related to gke/autopilot you can follow here: https://docs.splunk.com/Observability/gdi/opentelemetry/kubernetes-config.html#configure-google-kubernetes-engine-autopilot

Please let me know if you still run into issues.

**Comment by dmitryax** [comment]
@crcsmnky the GKE Autopilot support should be fixed in 0.67.0. Can you please try that version?

**Comment by atoulme** [comment]
Closing this issue as fixed, please reopen or create a new issue if more work is needed, thanks!
